### PR TITLE
control-service: handle null started by value

### PR DESCRIPTION
**Pull request by murphp15**
# Why
We are seeing errors when the UI when using the latest version of control service
# What
Write a test to check null values for startedby are supported. Also support null startedBy
# How was this tested
Locally

### CODE REVIEW

**Review by tozka — APPROVED** [review]

**Review by DeltaMichael — APPROVED** [review]

**Review by gorankokin — APPROVED** [review]

File: projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java [hunk 1]
```diff
@@ -109,7 +109,8 @@ public String startDataJobExecution(
       // We default to 'manual'; this also served for the purposes of backwards compatibility
       annotations.put(
           JobAnnotation.EXECUTION_TYPE.getValue(),
-          jobExecutionRequest.getStartedBy().contains("scheduled")
```
Comment (doks5) [inline projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:112]: Do we have a clue why the value is null, and not a username or some identifier? We hit this in an internal deployment of the Control Service, and it was unexpected to say the least.

Comment (murphp15) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:112]: The frontend doesn't send a startedBy. 
Is this not expected? If it is not expected we need to change the open api definition.

Comment (murphp15) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:112]: This code was changed recently. 
This caused the regression.

Comment (doks5) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:112]: Oopsie, yup, I saw the change. Should've `git blame`'ed the file first. Sorry

**Pull request merged**